src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 30px;
```

according to the design
```suggestion
  padding: 0 50px;
```